diaconn/src/main/java/info/nightscout/androidaps/diaconn/DiaconnG8Pump.kt

```diff
@@ -19,6 +19,9 @@ class DiaconnG8Pump @Inject constructor(
     private val aapsLogger: AAPSLogger,
     private val dateUtil: DateUtil
 ) {
+
+    var isPumplogUploadFailed: Boolean = false
```

Comment:
```suggestion
    var isPumpLogUploadFailed: Boolean = false
```
Please use camelCase